src/components/FileMenu/__tests__/GetLinkDialog.spec.js

```diff
@@ -25,27 +65,45 @@ describe('The FileMenu - GetLinkDialog component', () => {
 
     beforeEach(() => {
         shallowGetLinkDialog = undefined
-        props = {
-            type: 'visualization',
-            id: 'get-link-test-id',
-            onClose,
-        }
     })
 
     it('renders a Modal component', () => {
-        expect(getGetLinkDialogComponent(props).find(Modal)).toHaveLength(1)
+        expect(
+            getGetLinkDialogComponent({
+                type: tests[0].type,
+                id: tests[0].id,
+            }).find(Modal)
+        ).toHaveLength(1)
     })
 
-    it('renders a <a> tag containing the type and id props', () => {
-        const href = getGetLinkDialogComponent(props).find('a').prop('href')
+    tests.forEach((test) => {
```

Comment:
While this is fine, I just wanted to point out that Jest also has `test.each()` / `it.each()` for [these types of test scenarios](https://jestjs.io/docs/api#testeachtablename-fn-timeout). You could do something like this
```js
test.each(tests)(
    'renders an <a> tag with correct path and id for type $type',
    ({ type, baseUrl, id, expected}) => {
        // your test code
    }
)
```

Reply:
Refactored.